### PR TITLE
Update paraviewo and hdf5 to h5pp

### PR DESCRIPTION
**Pull request by zfergus**

### CODE REVIEW

**Review by teseoch — APPROVED** [review]

**Comment by zfergus** [comment]
```
eogram.lib(inflate.c.obj) : error LNK2005: inflate already defined in z.lib(zlib.dll)
geogram.lib(inflate.c.obj) : error LNK2005: inflateEnd already defined in z.lib(zlib.dll)
geogram.lib(inflate.c.obj) : error LNK2005: inflateInit_ already defined in z.lib(zlib.dll)
```

File: cmake/recipes/geogram.cmake [hunk 1]
```diff
@@ -41,7 +41,7 @@ include(FetchContent)
 FetchContent_Declare(
     polyfem_geogram
 	GIT_REPOSITORY https://github.com/polyfem/geogram.git
-	GIT_TAG        516c151c244d9019a9076a1a468d52a0f6dd195d
+	GIT_TAG        c790c30
```
Comment (zfergus) [inline cmake/recipes/geogram.cmake:44]: What about using the [official release](https://github.com/BrunoLevy/geogram)? I tried it on my mac and it compiles...

Comment (teseoch) [reply cmake/recipes/geogram.cmake:44]: the proble was windows, no?

Comment (zfergus) [reply cmake/recipes/geogram.cmake:44]: yeah, but I cannot test on windows

Comment (teseoch) [reply cmake/recipes/geogram.cmake:44]: Windows is fixed. we can use tags or revert the change

**Pull request merged**